test/unit/lib/cli/resolve-input.test.js

```diff
@@ -47,7 +47,7 @@ describe('test/unit/lib/cli/resolve-input.test.js', () => {
     });
 
     it('should not recognize -v with command', async () => {
```

Comment:
Should that test be named differently now?

Reply:
Good call, refactored a bit tests, so there's no confusion